scylla/tests/test_integration.py

```diff
@@ -0,0 +1,49 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
+from six import PY2
+
+from datadog_checks.base import AgentCheck
+from datadog_checks.scylla import ScyllaCheck
+
+from .common import (
+    FLAKY_METRICS,
+    INSTANCE_DEFAULT_METRICS,
+    INSTANCE_DEFAULT_METRICS_V2,
+)
+
+
+@pytest.mark.integration
+@pytest.mark.usefixtures('dd_environment')
+def test_instance_integration_check(aggregator, mock_db_data, dd_run_check, instance_legacy):
```

Comment:
nit, i would create a folder called `legacy` with all the legacy tests and use the file in the `tests` folder for OMv2 only